pkg/plugins/plugins.go

```diff
@@ -152,7 +152,7 @@ func (p *Plugin) activated() bool {
 }
 
 func (p *Plugin) activateWithLock() error {
-	c, err := NewClient(p.Addr, p.TLSConfig)
+	c, err := NewClientWithTimeout(p.Addr, p.TLSConfig, 30*time.Second)
```

Comment:
I don't think we can just set a 30s timeout here.
Many requests may take more than 30s (e.g. provisioning block storage for a volume).

This client gets cached and used for other requests, IIRC.